go/httpdispatcher/proxy.go

```diff
@@ -357,3 +373,126 @@ func (p *passthrough) checkDetectionMechanisms(ctx context.Context, e string) bo
 	}
 	return false
 }
+
+// HasSecurityEvents search for parking security event
+func (p *Proxy) HasSecurityEvents(ctx context.Context, mac string) bool {
+	securityEvent := false
+	var securityEventCount int
+	err := p.SecurityEvent.QueryRow(mac, 1).Scan(&securityEventCount)
```

Comment:
1 should be a constant for the default tenant id which will simplify a rewrite for multi-tenancy if we do one